src/Uno.UWP/Storage/Streams/FileOutputStream.cs

```diff
@@ -0,0 +1,33 @@
+#nullable enable
+
+using System;
+using System.IO;
+using Windows.Foundation;
+
+namespace Windows.Storage.Streams
+{
+	public sealed partial class FileOutputStream : IOutputStream, IDisposable, IStreamWrapper
+	{
+		private readonly Stream _stream;
+
+		internal FileOutputStream(string path, ulong position)
+		{
+			_stream = File.Open(path, FileMode.Open, FileAccess.Write, FileShare.Write);
```

Comment:
I think the FileShare should be `.None` or `.Read` here...
Doc https://docs.microsoft.com/en-us/dotnet/api/system.io.fileshare?view=netcore-3.1#fields

Reply:
Actually not, the share is controlled by the parent `RAStream`, and usually it has to be `ReadWrite` as we can create multiple `<In|Out>Stream` on the same `RAStream`